arches_her/settings.py

```diff
@@ -168,6 +200,11 @@ def get_optional_env_variable(var_name, default=None) -> str:
     "django_hosts",
     # "silk",
     "arches_her",
+    "django.contrib.postgres",
+    "arches_querysets",
+    "arches_component_lab",
+    "arches_controlled_lists",
```

Comment:
To be able to call `packages -o import_controlled_lists`, you'll need to move `arches_controlled_lists` above `arches`.

Here's an [example](https://github.com/archesproject/arches-lingo/blob/e17dee62616066ab7f90a63d51c1d79c4de6e351/arches_lingo/settings.py#L243-L270)

```suggestion
    "arches_her"
    "webpack_loader"
    ...
    "arches_controlled_lists",
    "arches"
    "arches.app.models",
    "arches.management",
    ...
```